app/views/cases/data_request_areas/new.html.slim

```diff
@@ -0,0 +1,33 @@
+- content_for :page_title do
+  = t('page_title.data_request_area_page', case_number: @case.number)
+
+- content_for :heading
+  = t('.heading')
+
+- content_for :sub_heading
+
+= link_to("Back", case_path(@case), class: 'govuk-back-link')
+
+span.visually-hidden
+  = t('common.case.header_case_number')
+  = @case.number
+
+= render partial: 'layouts/header'
+
+= GovukElementsErrorsHelper.error_summary @data_request_area,
+   "#{pluralize(@data_request_area.errors.count, t('common.error'))} #{ t('common.summary_error')}", ""
+
+= form_for @data_request_area, url: case_data_request_areas_path(@case) do |f|
+
+  = f.hidden_field :data_request_default_area
+  = f.radio_button_fieldset :data_request_area_type,
+    choices: DataRequestArea::data_request_area_types.keys do |fieldset|
+    - fieldset.radio_input('prison')
+    - fieldset.radio_input('probation')
+    - fieldset.radio_input('branston')
+    - fieldset.radio_input('branston_registry')
+    - fieldset.radio_input('mappa')
```

Comment:
Are these lines necessary? You're already passing in these values with `DataRequestArea::data_request_area_types.keys`